README.md

```diff
@@ -2,6 +2,10 @@
 
 This repository hosts the client iCommands, the default command line interface to iRODS.
 
+The client iCommands are released as a single package called `irods-icommands` in the packages.irods.org repository: https://packages.irods.org/
```

Comment:
We mention `packages.irods.org` twice. Consider doing one of the following:
- Change `in the packages.irods.org` to `in the iRODS packages`
- Make `packages.irods.org` into a link and remove `: https://packages.irods.org/`

Reply:
@trel already provided a suggestion.

Feel free to ignore this.

Reply:
I went with the other suggestion. Thanks